promscale/send-data/opentelemetry.md

```diff
@@ -50,20 +55,24 @@ service:
       receivers: [jaeger, otlp]
       exporters: [logging, otlp]
       processors: [batch]
+    metrics:
+      receivers: [otlp]
+      processors: [batch]
+      exporters: [prometheusremotewrite]
 ```
 
-Replace `<PROMSCALE_HOST>` and `<gRPC_PORT>` with the corresponding Promscale hostname and gRPC port. 
+Replace `<PROMSCALE_HOST>`, `<gRPC_PORT>` and `<HTTP_PORT>` with the corresponding Promscale hostname, gRPC port, and HTTP port. 
 For example, if you are running the OTLP Collector and the Promscale Connector on a 
 Kubernetes cluster the endpoint parameter would be similar to
-`endpoint: "promscale-connector.default.svc.cluster.local:9202"`
+`endpoint: "promscale-connector.default.svc.cluster.local:<PORT>"`
 
-## Send the data using OpenTelemetry instrumentation
+## Send data using OpenTelemetry instrumentation
 We recommend that you use the OpenTelemetry Collector to export data to an 
-observability backend in production as it offers batching, queued retries and many other functionalities by configuring the processors. However, you can also send data from 
+observability backend in production as it offers batching, queued retries, and many other functions by configuring the processors. However, you can also send data from 
```

Comment:
```suggestion
observability backend in production as it offers batching, queued retries, and many other functions by configuring the processors. However, you can also send data from 
```

We need to add guidance on hyphenation for words like backend to our word usage guide. For `backend` IBM says to hyphenate, Google says one word 🤷‍♀️